containers/layouts/ComposerModeRadios.js

```diff
@@ -27,7 +27,7 @@ const ComposerModeRadios = () => {
                 label={c('Label to change composer mode').t`Popup`}
                 onChange={handleChange(POPUP)}
             >
-                <img alt="Popup" src="/assets/img/design-system-website/popup.png" />
+                <img alt="Popup" src="assets/img/pm-images/composer-popup.svg" />
```

Comment:
Prefer to import it using the regular system

```
import composerPopupSvg from 'assets/img/pm-images/composer-popup.svg';
```
...

later in your component


```
<img alt="Popup" src={composerPopupSvg} />
```

Reply:
Ah, sure! Since I don't know what the folder structure will be eventually (namely what will be the path to fetch the images) I did not bother too much. But let me change it to the React standard to load images.